modules/cnv-connecting-resource-bridge-network.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-attaching-vm-multiple-networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge-based network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured and attached to the correct
+Network Interface Card on every node.
+* `vlan_filtering` must be enabled on the bridge if you want to use VLANs.
+
+.Procedure
+
+. Create a new file for the NetworkAttachmentDefinition in any local directory.
+The file must have the following contents, modified to match your
+configuration:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: a-bridge-network
+spec:
+  config: '{
+    "cniVersion": "0.3.0",
+    "name": "a-bridge-network", <1>
```

Comment:
maybe the number before the `,` something like this `"name": "a-bridge-network" <1>,`

Reply:
I think I prefer it the way it is. I think it looks a bit cleaner and is still understandable with the example that comes next. @kalexand-rh, are there guidelines about callouts that I should be following? The 4.0 contrib guide doesn't explicitly specify if they have to go at the end of the line or not.

Reply:
It's not explicitly called out anywhere, but after a little testing, I think it's an asciidoc limitation - the annotations can only go at the end of the line or they won't render.